<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DERIW42SLPC1327" ignored="0" name="Default test" tests="2" failures="1" timestamp="28 Jan 2020 16:25:10 GMT" time="0.519" errors="0">
  <testcase name="test1" time="0.046" classname="SamplePackage.FirstMavenClass"/>
  <testcase name="test2" time="0.374" classname="SamplePackage.FirstMavenClass">
    <failure type="java.lang.IllegalStateException" message="The driver executable does not exist: D:\Anand\AzureDevops\MavenWithAzureDevOps\GitToAzure\TestingAzure\FirstMavenProjectD:\Anand\AzureDevops\MavenWithAzureDevOps\GitToAzure\TestingAzure\FirstMavenProject\BrowserDrivers\chromedriver.exe">
      <![CDATA[java.lang.IllegalStateException: The driver executable does not exist: D:\Anand\AzureDevops\MavenWithAzureDevOps\GitToAzure\TestingAzure\FirstMavenProjectD:\Anand\AzureDevops\MavenWithAzureDevOps\GitToAzure\TestingAzure\FirstMavenProject\BrowserDrivers\chromedriver.exeat com.google.common.base.Preconditions.checkState(Preconditions.java:585)at org.openqa.selenium.remote.service.DriverService.checkExecutable(DriverService.java:146)at org.openqa.selenium.remote.service.DriverService.findExecutable(DriverService.java:141)at org.openqa.selenium.chrome.ChromeDriverService.access$000(ChromeDriverService.java:35)at org.openqa.selenium.chrome.ChromeDriverService$Builder.findDefaultExecutable(ChromeDriverService.java:159)at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:355)at org.openqa.selenium.chrome.ChromeDriverService.createDefaultService(ChromeDriverService.java:94)at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:123)at SamplePackage.FirstMavenClass.test2(FirstMavenClass.java:21)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- test2 -->
</testsuite> <!-- Default test -->
